recipes/libmysqlclient/all/conanfile.py

```diff
@@ -49,7 +51,8 @@ def _patch_files(self):
         os.rename(sources_cmake, sources_cmake_orig)
         os.rename("CMakeLists.txt", sources_cmake)
```

Comment:
Just some conventions

```suggestion
        tools.rename(sources_cmake, sources_cmake_orig)
        tools.rename("CMakeLists.txt", sources_cmake)
```